packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/index.js

```diff
@@ -2,17 +2,100 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { useCallback } from '@wordpress/element';
+import { useCallback, useMemo } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import { store as coreStore, useEntityBlockEditor } from '@wordpress/core-data';
+import { BlockEditorProvider, Inserter } from '@wordpress/block-editor';
 
 /**
  * Internal dependencies
  */
 import SidebarNavigationScreen from '../sidebar-navigation-screen';
-import NavigationInspector from '../navigation-inspector';
 import { useHistory } from '../routes';
+import NavigationMenuContent from './navigation-menu-content';
+import SidebarButton from '../sidebar-button';
+
+const NAVIGATION_MENUS_QUERY = { per_page: -1, status: 'publish' };
+
+function SidebarNavigationScreenWrapper( { children, actions } ) {
+	return (
+		<SidebarNavigationScreen
+			title={ __( 'Navigation' ) }
+			actions={ actions }
+			content={
+				<>
+					<p className="edit-site-sidebar-navigation-screen-navigation-menus__description">
+						{ __(
+							'Browse your site, edit pages, and manage your primary navigation menu.'
+						) }
+					</p>
+					{ children }
+				</>
+			}
+		/>
+	);
+}
 
 export default function SidebarNavigationScreenNavigationMenus() {
```

Comment:
I was forced to merge NavigationInspector into this component because the "BlockEditorProvider" now needs to wrap the `SidebarNavigationScreen` component (because the appender moved to the "actions" section there)